rc-node-bindings041.cc

```diff
@@ -0,0 +1,504 @@
+#include <nan.h>
```

Comment:
Please don't create a new file. We can obsolete support for the old API. I'd like to see what changed in this file.